Dockerfile

```
@@ -0,0 +1,37 @@
+# Use Node Iron LTS (20): https://nodejs.org/en/blog/release/v20.9.0
+FROM node:iron-bullseye-slim as wbs-dev-runner-base
```

```suggestion
FROM node:iron-bookworm-slim as wbs-dev-runner-base
```
bullseye is oldstable https://www.debian.org/releases/